lua/orgmode/objects/calendar.lua

```diff
@@ -184,14 +201,34 @@ function Calendar:render()
     table.insert(content, ' [i] - enter date')
   end
 
+  -- FIXME this line is currently not shown because of a bug.
```

Comment:
There is still a bug, that the following lines are not rendered. Currently don't understand why.

Reply:
This is now fixed.